version/constants.go

```diff
@@ -123,6 +124,21 @@ func init() {
 		}
 		RPCChainVMProtocolCompatibility[rpcChainVMProtocol] = versions
 	}
+
+	mainnetXChainStopVertexID, err := ids.FromString("jrGWDh5Po9FMj54depyunNixpia5PN4aAYxfmNzU8n752Rjga")
```

Comment:
Can you link to where you found these or are they not well represented on the explorers?

Reply:
I think putting it in the code would be good for documentation purposes tbh